src/main/webapp/feedback/interval-icon-feedback-count.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import TimestampUtil from '../timestamp-util.js';
+import IconFeedbackUtil from './icon-feedback-util.js';
+
+/** Stores IconFeeback counts in current interval. */
+export default class IntervalIconFeedbackCount {
+  /* Count of each IconFeedback type. */
+  #iconFeedbackCounts = {
+    [IconFeedbackUtil.TYPE_GOOD]: 0,
+    [IconFeedbackUtil.TYPE_BAD]: 0,
+    [IconFeedbackUtil.TYPE_TOO_FAST]: 0,
+    [IconFeedbackUtil.TYPE_TOO_SLOW]: 0,
+  }
+
+  /* Current interval IconFeedback counts appear in. */
+  #interval;
+
+  constructor(interval) {
+    this.#interval = TimestampUtil.timestampToString(interval);
+  }
+
+  /** Increments count of `iconFeedbackType` in dictionary. */
+  incrementIconFeedbackCount(iconFeedbackType) {
+    this.#iconFeedbackCounts[iconFeedbackType]++;
+  }
+
+  getIconFeedbackCounts() {
+    return this.#iconFeedbackCounts;
+  }
+
+  getInterval() {
```

Comment:
interval is a generic name, should it be timestampInterval or timeInterval?

Reply:
Fixed